### PR TITLE
api: Expose UCAN id

### LINKED ISSUE #364: Create DID API in Livepeer Studio
Add new endpoint here: https://github.com/livepeer/studio/tree/master/packages/api/src/controllers

The output format as specified in the Tech Design Doc.

### PR DESCRIPTION
**Pull request by red-0ne**
**What does this pull request do? Explain your changes. (required)**

Add `/api/did` endpoint to expose Livepeer's DID key got from UCAN_DID` env variable

**Does this pull request close any open issues?**

Fixes https://github.com/livepeer/catalyst/issues/364

### CODE REVIEW

File: packages/api/src/controllers/did.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Router from "express/lib/router";
+
+const app = Router();
+
+app.get("/", (_, res) => {
+  res.json({ did: process.env.UCAN_DID || "" });
```
Comment (leszko) [inline packages/api/src/controllers/did.js]: 1. I think the env variable name is not correct. In your PR https://github.com/livepeer/livepeer-infra/pull/1163, you defined it as `LP_API_DID`, here you use `UCAN_DID`. Could you double-check?

Also, please check how the environment variables are resolved in Livepeer Studio. I see that the `LP_API_` prefix is parsed somehow as flags, so please check if we should pass it somehow via flags to be consistent with other env variables.

2. I think that if the variable is not defined we should not return an empty string, but rather the specific HTTP Error code, I think `501` in this case.

3. I'd consider some simple validation if the env variable is in the correct format `did:key:...`.

**Review by leszko — CHANGES_REQUESTED** [review]
Thanks for the PR @red-0ne 

Added a few comments. PTAL. Saying that, I checked and it works fine 😃

File: packages/api/src/controllers/did.js (hunk 1 above)
Comment (victorges) [reply packages/api/src/controllers/did.js]: You should actually read that env from the parsed `req.config.ucanDid` instead! With the corresponding flag declaration in `parse-cli.ts` file. Those can also come from env vars (if you're curious check yargs usage), and in fact all of them do in our production deployments. Envs are also transformed from LP_API_SNAKE_CASE to lowerCamel

File: packages/api/src/store/experiment-table.ts [hunk 2]
```diff
@@ -7,8 +7,12 @@ import Table from "./table";
 import { WithID } from "./types";
 
 export async function isExperimentSubject(experiment: string, userId: string) {
-  const { audienceUserIds } = await db.experiment.getByNameOrId(experiment);
-  return audienceUserIds?.includes(userId) ?? false;
+  try {
```
Comment (leszko) [inline packages/api/src/store/experiment-table.ts]: Is this change related to your PR?

**Review by leszko — APPROVED** [review]
Added one comment. Other than that, LGTM 👍

File: packages/api/src/store/experiment-table.ts (hunk 2 above)
Comment (red-0ne) [reply packages/api/src/store/experiment-table.ts]: It came from a rebase from master.
